testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketConnectionAttemptTest.java

```diff
@@ -96,7 +95,7 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
         cb.handler(handler);
         cb.option(ChannelOption.ALLOW_HALF_CLOSURE, halfClosure);
-        ChannelFuture future = cb.connect(NetUtil.LOCALHOST, TestUtils.getFreePort()).awaitUninterruptibly();
+        ChannelFuture future = cb.connect(NetUtil.LOCALHOST, BAD_PORT).awaitUninterruptibly();
```

Comment:
Hum it would be preferred to crawl the ports but this is likely good enough. I guess put a `UNASSIGNED_PORT` constant in `TestUtils` (or something similar).